solutions/account-infrastructure-base/README.md

```diff
@@ -1,24 +1,23 @@
 # IBM Cloud Account Infrastructure Base solution
 
-An end-to-end deployable architecture solution that will provision the following:
-- A resource group.
-- A default set of FSCloud compliant Account Settings.
-- A Cloud Object Storage instance.
-- A Cloud Object Storage bucket encrypted with the KMS key.
-- An Activity Tracker target for the COS bucket
-- An Activity Tracker route to route events to the COS bucket.
-- A new Trusted Profile for Projects access.
+This is an end-to-end deployable architecture solution that provisions the following:
+- A resource group
+- A default set of Financial Services Cloud compliant account settings
+- An IBM Cloud Object Storage instance
+- An IBM Cloud Object Storage bucket encrypted with the key management service key
+- An Activity Tracker target for the Object Storage bucket
+- An Activity Tracker route to route events to the Object Storage bucket
+- A trusted profile to give Projects access to securely deploy this solution
```

Comment:
The trusted profile for projects is not for this solution, but for deploying other solutions in this account via that trusted profile so maybe something like `- A trusted profile to give Projects access to securely deploy solutions in this account`

Reply:
That sounds good, thank you! @MatthewLemmond